portfolio/src/main/webapp/script.js

```diff
@@ -126,19 +126,72 @@ function displayCommentSection() {
 }
 
 /** 
-  * Outputs JSON object as list on page. 
-  * @param {JSON} json The JSON to be formatted. 
+  * Outputs formatted comment section, where each comment 
+  * has a name and date to the left, text to the right, 
+  * and all comments are aligned vertically and stacked on top
+  * of one another. 
+  * @param {JSON} json The JSON representing the list of comment objects. 
   */
 function displayJSON(json) {
   const dataContainer = document.getElementById('comment-section');
-  let htmlToAdd = `<ul>`;
-  for (let i = 0; i < json.length; i++) {
-    htmlToAdd += `<li>${json[i]}</li>`;
+  htmlToAdd =`<table> <tr><td></td><td></td><td></td>` +
+    `<td></td><td></td><td></td><td></td></tr>`;
+
+  if (json.length === 0) {
+    htmlToAdd += 
+      `<tr rowspan="5">` +
+      ` <td colspan="7" id="be-the-first">` +
+          `Be the first to leave a comment!` +
+      ` </td>` +
+      `</tr>`;
   }
-  htmlToAdd += `</ul>`;
+  else {
+    let allCommentHTML = [...Array(json.length).keys()]
+      .map(i => getCommentHTML(json, i));
+    allCommentHTML.forEach(commentHTML => htmlToAdd += commentHTML);
+  }
+
+  htmlToAdd += `</table>`; 
   dataContainer.innerHTML = htmlToAdd; 
 }
 
+/**
+  * Returns the HTML representation of `json[i]`
+  * as a row in the comment section table.
+  * @param {JSON} json The JSON representing the list of all comments.
+  * @param {int} i The index of the desired comment.
+  * @return {String} The HTML representation. */
+function getCommentHTML(json, i) {
+  html =
+    `<tr>` +
+    ` <td class="comment-button"><button>X</button></td>` +
+    ` <td colspan = 2 class="user-info-box">` +
+    `   <b><u>${json[i].name}: </b></u><br>` +
+    `   <i class="comment-date">${prettyPrintTime(json[i].time)}</i>` +
+    ` </td>` +
+    ` <td colspan="4">` +
+        `${json[i].text}` +
+    ` </td>` +
+    `</tr>`;
+  return html;
+}
+
+/**
+  * Returns a time of the form: June 16, 5:30.
+  * @param {number} timeInmilliseconds The time in milliseconds since the 
+      epoch. 
+  * @return {String} The nicely-formatted string representing that time. 
+  */
+function prettyPrintTime(timeInMilliseconds) {
```

Comment:
I like the concept of pretty printing time :)

Instead of manually doing this work though, there are a lot of very configurable options in this library: https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Date/toDateString that would probably better suit your needs. 

Note re performance: Most notably, redefining the const months inside this method, which means it gets instantiated on every method call (waste of space). This is a good reason to prefer using JS standard libraries